### DISCUSSION TITLE
do not blindly rely on VSC jump to source it might open the wrong version file

### DISCUSSION THREAD

**Discussion by n4n0b1t3** [smartcontractkit/full-blockchain-solidity-course-py#1403] [General]
I was scratching my head for about 10 minutes until I realized that I was looking at the wrong version of a contract.
What happened was that I have my solidity remapping configured in VSC for a different version than under my project.

### example
In your projects `brownie-config.yaml` you have the following remapping:
`@openzeppelin=OpenZeppelin/openzeppelin-contracts@3.4.0`

and in VSC > solidity > remappings (which helps linting) you have this:
```json
 "solidity.remappings": [
        "@openzeppelin/=C:\\Users\\you\\.brownie\\packages\\OpenZeppelin\\openzeppelin-contracts@4.2.0",
]
```
### result
If you now ctrl+click on the class or function in your code, the code of the OpenZeppelin version 4.2.0 will be opened.
Just be aware of that, whenever you think something along the lines: "This can't be, this contract doesn't even have this method..."

In my case, it was the method _setTokenURI() which was "outsourced" in the newer versions.

**Comment by cromewar** [comment]
Yes Sir, also on the course the version used is 3.4.0 ;)

**Reply by ptn77** [reply]
hmmm...I did this too. What version should be used currently on Rinkeby?Can I still use the 3.4.0 version and change my VSC mapping back to 3.4.0? When I changed everything to 4.2.0 I get this error:
Status 404 when getting package versions from Github: 'Not Found'

Missing or forbidden.
If this issue persists, generate a Github API token and store it as the environment variable `GITHUB_TOKEN`:

What should the latest version be? I saw there's a version for 4.3.2 as well?
